pages/category.html

```diff
@@ -0,0 +1,114 @@
+<html>
+  <head>
+    <title>Category</title>
+  </head>
+
+  <body>
+    <!-- Logo-->
+    <div>
+      <a href="">
+        <img src="../img/HC Logo.svg" class="logo" alt="Logo">
+      </a>
+    </div>
+
+    <div class="banner-content">
+      <h1>HOTWAX COMMERCE</h1>
+      <h2>Omnichannel Order Management System for Retail Brands</h2>
+      <p>HotWax Commerce provides retailers with the best return on their inventory by facilitating omnichannel retailing strategies such as same-day Buy Online Pick-Up In Store (BOPIS), Ship From Store, and Pre-Orders in order to increase conversion rates and profitability.</p>
+    </div>
+
+    <!-- Shopify apps cards-->
+    <div>
+      <h2 class="card-header">Shopify apps</h2>
+
+      <div class="cards-container">
+        <div class="cards">
+          <div>
+            <a href="">
```

Comment:
use the anchor tag to wrap the div card. it should be the top most tag

Reply:
Done